flocker/node/agents/cinder.py

```diff
@@ -530,7 +530,7 @@ def list_volumes(self):
         http://docs.rackspace.com/cbs/api/v1.0/cbs-devguide/content/GET_getVolumesDetail_v1__tenant_id__volumes_detail_volumes.html
         """
         flocker_volumes = []
-        for cinder_volume in self.cinder_volume_manager.list(detailed=False):
+        for cinder_volume in self.cinder_volume_manager.list(detailed=True):
```

Comment:
Should it be the case where we use `False` for v1 instead of always `True` for `detailed`?